kymatio/scattering1d/tests/test_tensorflow_scattering1d.py

```diff
@@ -0,0 +1,65 @@
+import pytest
```

Comment:
Same comment as for NP: Need more tests. Add this to the existing issue (and rename it).

Reply:
For TF, do we also want to test in non-eager mode? Will the code even work there.

Reply:
I've added an issue